torch/_higher_order_ops/auto_functionalize.py

```diff
@@ -20,6 +21,12 @@
 )
 
 
+# regsiter_constant allows us to get a tree_spec from pytree.tree_flatten(FunctionSchema).
+# The tree_spec is proxable in the graph and we can get back the schema via
+# schema = pytree.tree_unflatten([], tree_spec)
+pytree.register_constant(torch.FunctionSchema)
```

Comment:
This should either:
1) go into torch._utils.pytree
2) we should have a wrapper class around torch.FunctionSchema that gets register_constant'ed

I'm not sure which is better